Sources_v3/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -82,17 +93,17 @@ class ChannelListController_Tests: StressTestCase {
         XCTAssertEqual(controller.state, .remoteDataFetchFailed(ClientError(with: error)))
     }
     
-    func test_noChangesAreReported_beforeCallingStartUpdating() throws {
+    func test_changesAreReported_beforeCallingsynchronize() throws {
```

Comment:
changes shouldn't be reported before calling synchronize

Reply:
Discussed.